### PR TITLE
Auto reload result page on state change from  scheduled to running

### PR DESCRIPTION
**Pull request by asdil12**
See https://progress.opensuse.org/issues/2238

### CODE REVIEW

File: assets/javascripts/test_result.js [hunk 1]
```diff
@@ -201,6 +201,16 @@ function setupResult(state, jobid) {
   if (state == "running" || state == "waiting") {
     setupRunning(jobid);
   }
+  else if (state == "scheduled") {
+    // reload when test starts
+    window.setInterval(function() {
+      $.ajax("/tests/" + jobid + "/status").done(function(newStatus) {
```
Comment (coolo) [inline assets/javascripts/test_result.js]: don't hard code URLs in javascript - I thought I told you yesterday

Comment (asdil12) [inline assets/javascripts/test_result.js]: changed.

**Comment by okurz** [comment]
Good approach

File: assets/javascripts/test_result.js [hunk 2]
```diff
@@ -199,7 +199,17 @@ function setupResult(state, jobid) {
...
   if (state == "running" || state == "waiting") {
-    setupRunning(jobid);
+    setupRunning(jobid, status_url, details_url);
+  }
+  else if (state == "scheduled") {
+    // reload when test starts
+    window.setInterval(function() {
+      $.ajax(status_url).done(function(newStatus) {
+        if (newStatus.state == 'running' || newStatus.state == 'waiting') {
+          setTimeout(function() {location.href = location.href.replace(/#.*/, '');}, 1000);
```
Comment (asdil12) [inline assets/javascripts/test_result.js]: done.

**Comment by coolo** [comment]
Now of course I wonder if this can't be tested.

**Comment by coolo** [comment]
it's not obvious how, but as this is not a critical feature put some thought into it

**Comment by asdil12** [comment]
Rebased.

**Comment by asdil12** [comment]
The complete js code on the running page is not tested because phantomJS doesn't like our websocket stuff. Also I don't see a way of updating the db that is used by the phantomJS mojo instance while the mojo instance is running.

**Pull request merged**